### ISSUE TITLE
Custom name for exports never reverts to default

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#1163]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=1163
Reporter:r...@thinkoomph.com Date:2011-08-26 09:40:12 Status:Duplicate Closed:2011-08-01

<b>What steps will reproduce the problem?</b>
1. select export for the first time
2. give export file a custom name
3. create the export

<b>What is the expected output? What do you see instead?</b>

I expect that next time I use the export tool, the default name of the export file will be the standard default, not the custom name I used last time. In addition, the default name shown is never the &quot;last used&quot;; instead it always default to the first custom name I ever used. 

In other words, every time I export a query, table, or db, the default filename is always preset to the first custom filename I ever created. It never updates based on the last one I used and never reverts to the program default.

<b>What version of Sequel Pro are you using? What version of MySQL are you</b>
<b>connecting to on the server?</b>

Version 0.9.9
